src/styles/reset.scss

```
@@ -0,0 +1,8 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+  box-sizing: border-box;
+}

```

Add box-sizing for all tag
```suggestion
}

* {
  box-sizing: border-box;
}

```